rules/0.17-allow-new-players

```diff
@@ -0,0 +1,29 @@
+import util
+
+# This value set to 0 until we figure out a good way to prevent dummy account abuse
+max_start_bonus = 0
+
+def should_allow(pr):
+  bonuses = pr.get_new_bonuses_or_raise()
+  
+  if len(bonuses) > 1:
+    raise Exception('Only one new player can be added in a PR')
+  elif len(bonuses) < 0:
+    raise Exception('Empty diff?')
+  
+  (points_user, points_name, points_change) = bonuses[0]
```

Comment:
What about replacing:

```
  if len(bonuses) > 1:
    raise Exception('Only one new player can be added in a PR')
  elif len(bonuses) < 0:
     raise Exception('Empty diff?')

  (points_user, points_name, points_change) = bonuses[0]
```

with just:

  (points_user, points_name, points_change), = bonuses

Reply:
Are you saying it will throw if there's not exactly one element? Even if so, it's less clear what happened.

Reply:
Yes, this throws if it can't destructure exactly as planned.  But ok if you don't like it.

Reply:
I could do a try statement, not sure if that's better. I just like the specificity of the failure messages.